snowpack/src/build/import-resolver.ts

```diff
@@ -126,31 +127,47 @@ export function createImportResolver({fileLoc, config}: {fileLoc: string; config
  * to a local file. These will additionally be transformed by the regular import resolver, so they do not need
  * to be finalized just yet
  */
-export function createImportGlobResolver({fileLoc, config}: {fileLoc: string; config: SnowpackConfig}) {
+export function createImportGlobResolver({
+  fileLoc,
+  config,
+}: {
+  fileLoc: string;
+  config: SnowpackConfig;
+}) {
+  const rootDir = path.parse(process.cwd()).root;
   return async function importGlobResolver(spec: string): Promise<string[]> {
+    let searchSpec = spec.replace(/\//g, path.sep);
```

Comment:
The original `spec` will always use `/`. But if we’re on Windows, let’s try and resolve using the native `path.sep`.